src/formatting_layer.rs

```diff
@@ -86,9 +87,23 @@ impl<W: for<'a> MakeWriter<'a> + 'static> BunyanFormattingLayer<W> {
             hostname: gethostname::gethostname().to_string_lossy().into_owned(),
             bunyan_version: 0,
             default_fields,
+            skip_fields: HashSet::new(),
```

Comment:
We should use https://lib.rs/crates/ahash as the hasher here, instead of the default one, to reduce the performance overhead of the check.

Reply:
will do